roles/openshift_node/tasks/upgrade.yml

```diff
@@ -45,6 +45,9 @@
   import_tasks: upgrade/rpm_upgrade_install.yml
   when: not openshift_is_atomic | bool
 
+# TODO(michaelgugino): Remove in 3.12
```

Comment:
Or leave it in and make it generic list of selinux tweaks we need to apply on both upgrade and install. I'm sure this won't be the last policy change we're asked to fix via openshift-ansible.

Reply:
@sdodson We can remove it in 3.12 because all upgrades since then will have included it.  We only need it for install in 3.12+.